scripts/generate-version-files.js

```diff
@@ -62,7 +62,8 @@ packageJsons.forEach(packageJsonPath => {
       `// ${packageJson.name}@${packageJson.version}
 // Do not modify this file, the file is generated as part of publish. The checked in version is a placeholder only.
 import { setVersion } from '@uifabric/set-version';
-setVersion('${packageJson.name}', '${packageJson.version}');`
+export const version = () => setVersion('${packageJson.name}', '${packageJson.version}');
```

Comment:
Suggestions; 
1. This should export a `setVersion` method. Calling it `version` is a little obscure. (rename the file too)
2. It shouldn't be called in this file.
3. Import setVersion and call it in the index file.

For partners that use `/lib/*` imports, they can just do this:
```tsx
import { setVersion } from 'oufr/lib/setVersion';
setVersion();
```
Then there are no side effects for them to consider.